cmd/cmd.go

```diff
@@ -82,16 +82,29 @@ func Main(embeddedWebsite fs.FS) {
 		timeline.Log.Fatal("could not start server", zap.Error(err))
 	}
 
-	// once the server is running, open GUI in web browser
-	if err := openWebBrowser("http://127.0.0.1:12002"); err != nil {
-		timeline.Log.Error("could not open web browser", zap.Error(err))
+	if isDesktopAvailable() {
+		// once the server is running, open GUI in web browser
+		if err := openWebBrowser("http://127.0.0.1:12002"); err != nil {
+			timeline.Log.Error("could not open web browser", zap.Error(err))
+		}
 	}
 
 	if startedServer {
 		select {}
 	}
 }
 
+func isDesktopAvailable() bool {
+	if runtime.GOOS == "linux" {
+		available := os.Getenv("DISPLAY") != ""
+		if !available {
+			timeline.Log.Warn("DISPLAY environment variable not set, not opening web browser")
```

Comment:
What if we don't log in this function, but log something like this above where we actually decide what to do with the information?

Reply:
Sounds good! I'll take care of it.

Reply:
@rubiojr Do you want me to jump on it? I might have a chance in the next few days if that'll help.

Reply:
Feel free! Imma be AFK a few days.